src/services/info/notice/modules/NoticeList.vue

```diff
@@ -5,7 +5,7 @@
                        @change="handleToolboxChange"
             >
                 <template #left-area>
-                    <p-select-dropdown :items="dropdownItems" :selected.sync="selectedScope" />
+                    <p-select-dropdown :items="dropdownItems" :selected.sync="selectedScope" @update:selected="hadleSearchScopeChange" />
```

Comment:
```suggestion
                    <p-select-dropdown :items="dropdownItems" :selected="selectedScope" @update:selected="hadleSearchScopeChange" />
```
update:selected를 사용하고 있다보니 `.sync`를 제거하시고 `handleSearchScopeChange`에서 받아온 값을 selectedScope에 할당하면 좋을 것 같습니다!
(관련 문법은 [여기](https://v2.vuejs.org/v2/guide/components-custom-events.html#sync-Modifier)를 참고해주세요!)